### ISSUE TITLE
both 31 mb and 3 mb versions broken

### ISSUE THREAD

**Issue by hr1juldey** [jpbouza/BlenRig#1]
sir I am using blender 3.1 caustics build with blenrig 6 and 5 both are showing error but in blender 3.0 it is behaving fine.
what to do??
showing index out of reach error

**Comment by jpbouza** [comment]
Hi!

When are you getting that error? Is it when you enable the addon or when you do something specific with it?

**Comment by hr1juldey** [comment]
During the shift A phase

**Comment by jpbouza** [comment]
You mean when you add the blenrig armature in a new scene?

**Comment by hr1juldey** [comment]
Correct.  I get the widgets or the face mesh but nothing from the armature

**Comment by jpbouza** [comment]
@hr1juldey I just downloaded Blender 3.1 and the addon from here, I have no issues. 

From what you said, you have BlenRig 5 installed also? Could you try removing both BlenRig 5 and 6, then just installing BlenRig 6?

You are definitely not supposed to have both addons installed, BlenRig 6 already supports the old rigs.

**Comment by hr1juldey** [comment]
Ok, I'll check it out.
